### DISCUSSION TITLE
Colors of charts (history and statistics): watch out for the colorblind

### DISCUSSION THREAD

**Discussion by dougiteixeira** [home-assistant/frontend#11054] [Accessibility]
The colors defined for the chart data have small shade variations that confuse people with color blindness (even people with a mild degree).

It would be good for us who are color blind if the color tones were more vibrant or if it was open to customizing the color of each entity in the chart (or both).

Statistics Graph Card:
![image](https://user-images.githubusercontent.com/31328123/147797649-803d9a7f-fb43-4022-9229-c1ebc61f4e6d.png)

History Graph Card:
![image](https://user-images.githubusercontent.com/31328123/147797700-a42f4a05-07d8-45a1-98e0-b4879da75e44.png)

It seems unlikely, but people with color blindness have difficulty distinguishing between different shades of blue in the examples above. Even shades of orange are not easy. Sorry if I didn't even get the colors of the tones right...

**Comment by spacegaier** [comment]
Since 2021.12 those colors can be themed: https://github.com/home-assistant/frontend/pull/10698
